<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Participant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Participant 1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Participant 9</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="participant1"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="participant_9">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Participant 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Participant 10</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="participant_2"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="participant_10">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Participant 3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Participant 11</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="participant_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="participant_11">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Participant 4</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" rowspan="2">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Participant 12</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" rowspan="2">
          <widget class="QLineEdit" name="participant_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="participant_22">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Participant 5</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1" rowspan="2">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Participant 13</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" rowspan="2">
          <widget class="QLineEdit" name="participant_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="participant_23">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Participant 6</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Participant 14</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLineEdit" name="participant_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="participant_24">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="14" column="0" rowspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Participant 7</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Participant 15</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1" rowspan="2">
          <widget class="QLineEdit" name="participant_25">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLineEdit" name="participant_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="17" column="0" rowspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Participant 8</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Participant 16</string>
           </property>
          </widget>
         </item>
         <item row="18" column="1" rowspan="2">
          <widget class="QLineEdit" name="participant_26">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLineEdit" name="participant_8">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rb_ping_pong">
          <property name="text">
           <string>Ping Pong</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_uno">
          <property name="text">
           <string>Uno</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_pool">
          <property name="text">
           <string>Pool</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>Load Participans</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
